app/src/main/res/values-de/strings.xml

```diff
@@ -583,7 +583,7 @@
     <string name="signature_is_valid">Die Signatur ist gültig</string>
     <string name="signature_is_invalid">Die Signatur ist ungültig</string>
     <string name="no_signature_available">Keine Signatur verfügbar</string>
-    <string name="subscribed_to">"Abonniert bei "</string>
+    <string name="subscribed_to">"Abonniert zu "</string>
```

Comment:
The zu is redundant since it shows who you just subscribed to.
So the full message in the app would be:

$channel_name$ abonniert.